jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/ZipUtilTest.java

```diff
@@ -66,6 +70,28 @@ public void testZipSlipVulnerability_unix() throws URISyntaxException {
     verifyZipSlipSafe(archive);
   }
 
+  @Test
+  public void testUnzip_modificationTimePreserved() throws URISyntaxException, IOException {
+    Path destination = tempFolder.getRoot().toPath();
+    Path archive =
+        Paths.get(Resources.getResource("plugins-common/test-archives/test.zip").toURI());
+    try (ZipFile zipFile = new ZipFile(archive.toString())) {
+      FileTime sourceModificationTime1 = zipFile.getEntry("file1.txt").getLastModifiedTime();
+      FileTime sourceModificationTime2 = zipFile.getEntry("my-zip/file2.txt").getLastModifiedTime();
+      FileTime sourceModificationTime3 =
+          zipFile.getEntry("my-zip/some/sub/folder/file3.txt").getLastModifiedTime();
```

Comment:
I think, in this case, we should just have these values as constants that we know. Manually look in the zip and grab the values, and use those here. That will help us determine if some other part of the machinery breaks (like getLastModifiedTime).

Reply:
`ZipFile.getEntry().getLastModifiedTime()` and `Files.getLastModifiedTime()` are standard JDK. We can trust them, so I think it's okay?

Reply:
Right, I don't actually expect those to break. But in a case that we have access to the constant values (as it is a premade zip in the repository?), I feel like it's a more robust test style?